Task4MathFormulas/Task4MathFormulas/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task4MathFormulas
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Задачa 21:\r\n");
+            Console.WriteLine("Введите значение X:\r\n");
```

Comment:
Лишняя пустая строка на консоли усложняет восприятие.
```suggestion
            Console.WriteLine("Введите значение X:");
```